# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/svg/SVGAnimatedRect.cpp
# Commit: b848ca90888e
# Full Hash: b848ca90888ee8e446f3d54b65c3ef68c71a1451
# Author: longsonr <longsonr@gmail.com>
# Date: 2019-06-20 22:06:31
# Regressor Bug: 1551030
# File Overlap Count: 2
# Description:
#   Bug 1551030 - Merge all SVGRect classes r=dholbert
# ==============================================================================

diff -r 20956a783e08 -r b848ca90888e dom/svg/SVGAnimatedRect.cpp
--- a/dom/svg/SVGAnimatedRect.cpp	Thu Jun 20 14:19:57 2019 +0300
+++ b/dom/svg/SVGAnimatedRect.cpp	Thu Jun 20 07:03:54 2019 -0700
@@ -7,8 +7,8 @@
 #include "SVGAnimatedRect.h"
 #include "mozilla/dom/SVGAnimatedRectBinding.h"
 #include "mozilla/dom/SVGElement.h"
+#include "mozilla/dom/SVGRect.h"
 #include "SVGAnimatedViewBox.h"
-#include "SVGIRect.h"
 
 namespace mozilla {
 namespace dom {
@@ -26,11 +26,11 @@
   SVGAnimatedViewBox::sSVGAnimatedRectTearoffTable.RemoveTearoff(mVal);
 }
 
-already_AddRefed<SVGIRect> SVGAnimatedRect::GetBaseVal() {
+already_AddRefed<SVGRect> SVGAnimatedRect::GetBaseVal() {
   return mVal->ToDOMBaseVal(mSVGElement);
 }
 
-already_AddRefed<SVGIRect> SVGAnimatedRect::GetAnimVal() {
+already_AddRefed<SVGRect> SVGAnimatedRect::GetAnimVal() {
   return mVal->ToDOMAnimVal(mSVGElement);
 }
 